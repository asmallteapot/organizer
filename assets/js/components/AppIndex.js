import React from 'react'
import { Divider, Paper } from 'material-ui'

export default (props) => (
  <Paper zDepth={1} className="app-splash">
    <h1>East Bay Forward <div className="pop">Organizer</div></h1>
    <p>The housing shortage is not an unintended policy failure. The Bay Area
      has a housing shortage because of decades of voting and organizing against
      housing. The solution is to organize for housing.</p>
    <p>East Bay Forward Organizer helps people organize for housing.</p>
    <Divider />
    <p />
    <p><a href={SLACK_LOGIN_URL}>
      <img
        alt="Sign in with Slack"
        height="40"
        width="172"
        src="https://platform.slack-edge.com/img/sign_in_with_slack.png"
        srcSet="https://platform.slack-edge.com/img/sign_in_with_slack.png 1x, https://platform.slack-edge.com/img/sign_in_with_slack@2x.png 2x"
      />
    </a></p>
    <p>Please sign in with East Bay Forward Slack to continue.</p>
    <p className="github-link"><a href="https://github.com/tdfischer/organizer/"><i className="fa fa-github"/></a></p>
  </Paper>
)
