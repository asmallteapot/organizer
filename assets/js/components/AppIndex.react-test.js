import React from 'react'
import { mount } from 'enzyme'
import AppIndex from './AppIndex'

test('Smoke test', () => {
  global.SLACK_LOGIN_URL = "";
  const component = mount(<AppIndex />);
  expect(component.html()).toMatchSnapshot()
})
