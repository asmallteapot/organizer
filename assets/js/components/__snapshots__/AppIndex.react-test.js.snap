// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Smoke test 1`] = `"<div class=\\"app-splash\\"><h1><!-- react-text: 3 -->East Bay Forward <!-- /react-text --><div class=\\"pop\\">Organizer</div></h1><p>The housing shortage is not an unintended policy failure. The Bay Area has a housing shortage because of decades of voting and organizing against housing. The solution is to organize for housing.</p><p>East Bay Forward Organizer helps people organize for housing.</p><p><a href=\\"\\"><img alt=\\"Sign in with Slack\\" height=\\"40\\" width=\\"172\\" src=\\"https://platform.slack-edge.com/img/sign_in_with_slack.png\\" srcset=\\"https://platform.slack-edge.com/img/sign_in_with_slack.png 1x, https://platform.slack-edge.com/img/sign_in_with_slack@2x.png 2x\\"></a></p><p>Please sign in with East Bay Forward Slack to continue.</p><p class=\\"github-link\\"><a href=\\"https://github.com/tdfischer/organizer/\\"><i class=\\"fa fa-github\\"></i></a></p></div>"`;
